Lib/json/tool.py

```diff
@@ -48,6 +50,8 @@ def main():
         dump_args['indent'] = None
         dump_args['separators'] = ',', ':'
 
+    with_colors = can_colorize()
+
```

Comment:
This variable isn't doing much, shall we just call the function directly?

Reply:
I may have been too cautious here. Without the variable, the function would be called in a loop and I was worried the result could change in between invocations, though that is really unlikely. If you prefer I can remove it?

Reply:
Actually, I applied your other suggestion which already removed it so never mind :smile: